.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+# Description
+
+Please include a summary of the changes i.e. Java version updates, libraries patched, or issues added to the policy file.
+
+# Checklist:
+
+- [ ] I have ensured the Defra version in the [JOB.env](../JOB.env) file matches that in the [Dockerfile](../Dockerfile)
```

Comment:
I discovered the relative links don't work on the PR page, so have removed them on the latest Node.js version